environments.json

```diff
@@ -2603,370 +2603,431 @@
     "family": "V8",
     "equals": "chrome57",
     "short": "OP 44",
+    "release": "2017-03-21",
     "obsolete": "very"
   },
   "opera45": {
     "full": "Opera 45",
     "family": "V8",
     "equals": "chrome58",
     "short": "OP 45",
+    "release": "2017-05-10",
     "obsolete": "very"
   },
   "opera46": {
     "full": "Opera 46",
     "family": "V8",
     "equals": "chrome59",
     "short": "OP 46",
+    "release": "2017-06-22",
     "obsolete": "very"
   },
   "opera47": {
     "full": "Opera 47",
     "family": "V8",
     "equals": "chrome60",
     "short": "OP 47",
+    "release": "2017-08-09",
     "obsolete": "very"
   },
   "opera48": {
     "full": "Opera 48",
     "family": "V8",
     "equals": "chrome61",
     "short": "OP 48",
+    "release": "2017-09-27",
     "obsolete": "very"
   },
   "opera49": {
     "full": "Opera 49",
     "family": "V8",
     "equals": "chrome62",
     "short": "OP 49",
+    "release": "2017-11-08",
     "obsolete": "very"
   },
   "opera50": {
     "full": "Opera 50",
     "family": "V8",
     "equals": "chrome63",
     "short": "OP 50",
+    "release": "2018-01-04",
     "obsolete": "very"
   },
   "opera51": {
     "full": "Opera 51",
     "family": "V8",
     "equals": "chrome64",
     "short": "OP 51",
+    "release": "2018-02-07",
     "obsolete": "very"
   },
   "opera52": {
     "full": "Opera 52",
     "family": "V8",
     "equals": "chrome65",
     "short": "OP 52",
+    "release": "2018-03-14",
     "obsolete": "very"
   },
   "opera53": {
     "full": "Opera 53",
     "family": "V8",
     "equals": "chrome66",
     "short": "OP 53",
+    "release": "2018-05-10",
     "obsolete": "very"
   },
   "opera54": {
     "full": "Opera 54",
     "family": "V8",
     "equals": "chrome67",
     "short": "OP 54",
+    "release": "2018-06-28",
     "obsolete": "very"
   },
   "opera55": {
     "full": "Opera 55",
     "family": "V8",
     "equals": "chrome68",
     "short": "OP 55",
+    "release": "2018-08-16",
     "obsolete": "very"
   },
   "opera56": {
     "full": "Opera 56",
     "family": "V8",
     "equals": "chrome69",
     "short": "OP 56",
+    "release": "2018-09-25",
     "obsolete": "very"
   },
   "opera57": {
     "full": "Opera 57",
     "family": "V8",
     "equals": "chrome70",
     "short": "OP 57",
+    "release": "2018-11-28",
     "obsolete": "very"
   },
   "opera58": {
     "full": "Opera 58",
     "family": "V8",
     "equals": "chrome71",
     "short": "OP 58",
-    "obsolete": "very"
-  },
-  "opera59": {
-    "full": "Opera 59",
-    "family": "V8",
-    "equals": "chrome72",
-    "short": "OP 59",
+    "release": "2019-01-23",
     "obsolete": "very"
   },
   "opera60": {
     "full": "Opera 60",
     "family": "V8",
     "equals": "chrome73",
     "short": "OP 60",
+    "release": "2019-04-09",
     "obsolete": "very"
   },
-  "opera61": {
-    "full": "Opera 61",
-    "family": "V8",
-    "equals": "chrome74",
-    "short": "OP 61",
-    "obsolete": true
-  },
```

Comment:
and opera 61?

Reply:
According to my [source](https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser#Opera_2019), version 61 was skipped.